frontend/src/container/BillingContainer/BillingUsageGraph/BillingUsageGraph.tsx

```diff
@@ -46,6 +46,18 @@ const calculateStartEndTime = (
 ): { startTime: number; endTime: number } => {
 	const timestamps: number[] = [];
 	data?.details?.breakdown?.forEach((breakdown: any) => {
+		if (breakdown?.dayWiseBreakdown?.breakdown?.length === 1) {
```

Comment:
add inline comment for the reasoning why are we doing this. it won't be trivial for someone to understand looking at this later.